tests/ssl-opt.sh

```diff
@@ -210,6 +208,7 @@ skip_next_test() {
 requires_config_enabled() {
     case $CONFIGS_ENABLED in
         *" $1 "*) :;;
+        *" $1="*) :;;
```

Comment:
Minor: this can be written as
```suggestion
        *" $1"[\ =]*) :;;
```
and likewise below.